core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -328,5 +378,18 @@ object Props extends Logger {
         Map()
     }
   }
+
+  private var prepended:List[java.util.Map[Object, Object]] = Nil
+  private var appended:List[java.util.Map[Object, Object]] = Nil
+  private var interpolators:List[java.util.Map[Object, Object]] = Nil
+
+  /**
+   * Resets the Props stacks to allow for isolated unit tests
+   */
+  private [util] def testReset() = {
+    prepended = Nil
+    appended = Nil
+    interpolators = Nil
+  }
```

Comment:
Is there something else we could do here aside from adding code to the mainline branch that's only ever going to be used in tests?